src/Parser/GpxParser.js

```diff
@@ -8,7 +8,7 @@ import * as THREE from 'three';
 import Line from 'three.meshline';
 import Coordinates from '../Core/Geographic/Coordinates';
 import Capabilities from '../Core/System/Capabilities';
-import { patchMaterialForLogDepthSupport } from '../Provider/3dTilesProvider';
+import { patchMaterialForLogDepthSupport } from './B3dmParser';
```

Comment:
While you are at moving patchMaterialForLogDepthSupport, it does not look like it is more specific to B3DM than to 3Dtiles as it is used by GPX. Should not it be placed in a more generic location ?

Reply:
Done. Moved it to a *ShaderUtils* file.